### PR TITLE
test: component repository and image tags are configurable now

### PR DESCRIPTION
**Pull request by j4k4**
## Configuration
Test components are docker container like wiremock or localstack which support you in your integration tests to mock
your applications infrastructure.

#### image configuration
The used image for a component can be configured on a global and on a per component level. The following configuration
snipped shows a config for two wiremock components. **mockA** gets the image configuration from the global level which is
configured via the key `test.defaults.images.wiremock` and **mockB** has local configuration. With this, **mockA** gets
tag `3.4.0` and **mockB** gets `3.3.0`.

```yaml
test:
  defaults:
    images:
      wiremock:
        repository: wiremock/wiremock
        tag: 3.4.0
  components:
    wiremock:
      mockA:
        expire_after: 5m
      mockB:
        image:
          repository: wiremock/wiremock
          tag: 3.3.0
        expire_after: 5m
```

### CODE REVIEW

File: pkg/test/env/config.default.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        tag: "8.0"
+      redis:
+        repository: "redis"
+        tag: "7-alpine"
+      s3:
+        repository: "minio/minio"
+        tag: "RELEASE.2023-03-09T23-16-13Z"
+      wiremock:
+        repository: "wiremock/wiremock"
+        tag: "3.3.1-alpine"
```
Comment (applike-ss) [inline pkg/test/env/config.default.yml:21]: please change to non-alpine to ensure arm64 compatibility

File: pkg/test/env/environment.go [hunk 2]
```diff
@@ -212,7 +233,7 @@ func (e *Environment) LoadFixtureBuilderFactories(factories ...fixtures.FixtureB
 			return fmt.Errorf("can not build fixture builder: %w", err)
 		}
 
-		if err = e.fixtureLoader.Load(e.ctx, fixtureBuilder.Fixtures()); err != nil {
+		if err := e.fixtureLoader.Load(e.ctx, fixtureBuilder.Fixtures()); err != nil {
```
Comment (applike-ss) [inline pkg/test/env/environment.go:236]: why re-create this var? in line 229 you already have an err var defined.

**Review by applike-ss — CHANGES_REQUESTED** [review]

File: pkg/test/env/environment.go (hunk 2 above)
Comment (j4k4) [reply pkg/test/env/environment.go:236]: sometimes the linter complains about these things due to: https://go-critic.com/overview.html#sloppyreassign  
you get something like:
```
sloppyReassign: re-assignment to `err` can be replaced with `err := env.runner.RunContainers(skeletons)` (gocritic)
```

File: pkg/test/env/config.default.yml (hunk 1 above)
Comment (j4k4) [reply pkg/test/env/config.default.yml:21]: changed to `3.4.1`

File: pkg/test/env/environment.go (hunk 2 above)
Comment (applike-ss) [reply pkg/test/env/environment.go]: strange, my goland often complains for these when i re-assign a var that exists already. But fine with me.

**Review by applike-ss — APPROVED** [review]

**Pull request merged**